FAQ/How to Read an Existing Excel File.md

```diff
@@ -0,0 +1,42 @@
+# How to Read an Existing Excel File
+
+```powershell
+Import-Module ImportExcel
```

Comment:
Don't need the `Import-Module`, once `ImportExcel` is installed, it will be autoloaded by PowerShell.

Reply:
Thanks for the feedback! I have made this change. I also added another header to match the styling of the other documents.